blog/PyMC_Past_Present_Future.md

```diff
@@ -0,0 +1,166 @@
+
+### **2011-2015**
+
+The Metropolis-Hastings and Gibbs samplers, two algorithms used to draw samples from a probability distribution, performed slowly for large or complex models.  
+
+The next generation of bayesian inference methods aimed to solve these problems — namely, a gradient-based MCMC or hybrid Monte Carlo. 
+
+![Gradient Based MCMC](../_static/chris-pymcon-keynote/2011_2015_Gradient_based_MCMC.png)
+
+The No-U-Turn-Sampler (NUTS algorithm), [developed by Matt Hoffman and Andrew Gelman in 2011](https://arxiv.org/abs/1111.4246), used information about the gradient of the log posterior-density to identify regions of higher probability to help it converge quickly on large problems. This was much faster than traditional sampling methods.
+
+John Salvatier developed the mcex package to experiment with gradient-based MCMC samples, and the following year he was invited by the team to re-engineer PyMC. Rather than relying on Fortran, he did this using the Theano package — a deep learning library initially developed for implementing neural network models.
+
+![Theano](../_static/chris-pymcon-keynote/Theano.png)
+
+The use of Theano allowed for constructing and compiling a graph to C. This enabled optimization, use on GPUs, and automatic differentiation. 
+
+Later on several other methods were added to PyMC, such as variational inference methods, largely thanks to the efforts of Taku Yoshioka and Max Kucherov.
+
+![PyMC3 in the Wild](../_static/chris-pymcon-keynote/PyMC3_in_Wild.png)
+
+There was a significant increase in the number of users of Pymc 3.0 compared to Version 2. 
+
+Some examples of notable users include Aaron McNeil, a marine biologist who used PyMC to model the spread of crown of thorns starfish on the Great Barrier Reef, and Kevin Systrom and Thomas Vladick, the founder of Instagram and a developer, respectively, who used PyMC to model the basic reproductive number (Rt) of COVID-19 in different states in the US. This allowed them to provide real-time information about the spread of the virus.
+
+### **2016**
+
+![NumFocus_LRG.png](../_static/chris-pymcon-keynote/NumFocus_LRG.png)
+
+In 2016, [PyMC became a sponsored project under NumFocus](https://numfocus.org/uncategorized/numfocus-announces-new-fiscally-sponsored-project-pymc3), a nonprofit organization that provides sustainability and support to open-source projects. By joining NumFocus, PyMC was able to access educational programs and events, as well as additional resources, to help ensure its continued development and success. 
+
+This support has been instrumental in allowing PyMC to continue to grow and thrive in the Open Source space, as well as in expanding community and diversity efforts.
+
+### **2017-2020**
+
+In October 2017, the team behind Theano, a deep learning framework, decided to discontinue the project due to the availability of other well-supported frameworks. 
```

Comment:
Suggested change: "... the Mila team at the University of Montreal responsible for the Theano framework, used by PyMC as its computational backend, decided ..."